core/src/main/java/com/scalar/db/storage/dynamo/DynamoTableMetadataManager.java

```diff
@@ -51,7 +72,7 @@ public TableMetadata getTableMetadata(Operation operation) {
 
   @Override
   public TableMetadata getTableMetadata(String namespace, String table) {
-    String fullName = namespace + "." + table;
+    String fullName = Utility.getFullTableName(namespacePrefix, namespace, table);
```

Comment:
Maybe we can statically import `Utility.getFullTableName` to make the code simpler and shorter.